cmd/podman/build.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	buildFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "build-arg",
+			Usage: "`argument=value` to supply to the builder",
+		},
+		cli.StringSliceFlag{
+			Name:  "file, f",
+			Usage: "`pathname or URL` of a Dockerfile",
+		},
+		cli.StringFlag{
+			Name:  "format",
+			Usage: "`format` of the built image's manifest and metadata",
+		},
+		cli.BoolFlag{
+			Name:  "pull-always",
+			Usage: "pull the image, even if a version is present",
+		},
+		cli.BoolFlag{
+			Name:  "quiet, q",
+			Usage: "refrain from announcing build instructions and image read/write progress",
+		},
+		cli.StringFlag{
+			Name:  "runtime",
+			Usage: "`path` to an alternate runtime",
+		},
+		cli.StringSliceFlag{
+			Name:  "runtime-flag",
+			Usage: "add global flags for the container runtime",
+		},
+		cli.StringFlag{
+			Name:  "signature-policy",
+			Usage: "`pathname` of signature policy file (not usually used)",
+		},
+		cli.StringSliceFlag{
+			Name:  "tag, t",
+			Usage: "`tag` to apply to the built image",
+		},
+		cli.BoolFlag{
+			Name:  "tls-verify",
+			Usage: "Require HTTPS and verify certificates when accessing the registry",
+		},
+	}
+	buildDescription = "This command launches the Buildah command to build an OCI Image. Buildah must be installed for this command to work."
+	buildCommand     = cli.Command{
+		Name:        "build",
+		Aliases:     []string{"build"},
+		Usage:       "Build an image using instructions in a Dockerfile",
+		Description: buildDescription,
+		Flags:       buildFlags,
+		Action:      buildCmd,
+		ArgsUsage:   "CONTEXT-DIRECTORY | URL",
+	}
+)
+
+func buildCmd(c *cli.Context) error {
+
+	budCmdArgs := []string{"bud"}
+
+	if c.IsSet("build-arg") {
+		budCmdArgs = append(budCmdArgs, "--build-arg")
+		budCmdArgs = append(budCmdArgs, c.StringSlice("build-arg")...)
```

Comment:
I think this has to be a for loop, no need to check IsSet.
for _, barg := range c.StringSlice("buildah-arg) {
		budCmdArgs = append(budCmdArgs, "--build-arg", barg) 
}

Reply:
Hmm, I think you're right.  I'll have to check the test to see if I can add something there as it didn't trip up as is.